tests/python_package_test/test_sklearn.py

```diff
@@ -4,24 +4,37 @@
 import math
 import os
 import unittest
-import warnings
 
 import lightgbm as lgb
 import numpy as np
+import pytest
 from sklearn import __version__ as sk_version
 from sklearn.base import clone
 from sklearn.datasets import load_svmlight_file, make_multilabel_classification
-from sklearn.exceptions import SkipTestWarning
 from sklearn.metrics import log_loss, mean_squared_error
 from sklearn.model_selection import GridSearchCV, RandomizedSearchCV, train_test_split
 from sklearn.multioutput import (MultiOutputClassifier, ClassifierChain, MultiOutputRegressor,
                                  RegressorChain)
-from sklearn.utils.estimator_checks import (_yield_all_checks, SkipTest,
-                                            check_parameters_default_constructible)
 from sklearn.utils.validation import check_is_fitted
 
+from sklearn.utils.estimator_checks import check_parameters_default_constructible
```

Comment:
Please move this line before `from sklearn.utils.validation ...` to keep alphabetical order for imports.